app/src/main/java/com/neo/speaktouch/utils/NodeValidator.kt

```diff
@@ -1,7 +1,7 @@
 /*
- * Node focus and read validations.
+ * [FILE DESCRIPTION].
  *
- * Copyright (C) 2023 Irineu A. Silva.
+ * Copyright (C) 2023 [AUTHOR].
```

Comment:
I saw this just now. Something like this couldn't have been overlooked. This class is one of, if not the most important in the app

Reply:
I will correct it directly in develop.